python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -125,16 +123,15 @@ def _computePsfImage(self, position):
             # data in some bands, so we track this error to distinguish it
             # from unknown errors.
             msg = "Failed to compute PSF at {} in band {}"
-            raise IncompleteDataError(msg.format(position, self.filters[bidx]))
+            raise IncompleteDataError(msg.format(position, filters[bidx]))
```

Comment:
Add `from None` (if you want the traceback to end here) or `from err` (with `as err` in the `except` earlier) to avoid extra "while handling this exception...another exception" noise in the traceback.